org.lflang/src/org/lflang/generator/ts/TSReactionGenerator.kt

```diff
@@ -119,27 +132,29 @@ class TSReactionGenerator(
                 }
             }
         }
+        reactCode = reaction.code.toText()
```

Comment:
This variable also seems unnecessary...

Reply:
Thank you. I deleted the unnecessary variable! https://github.com/lf-lang/lingua-franca/pull/1318/commits/f327feef4c499641d83833eb1fd3591f20c35834